src/sidebar/store/modules/sidebar-panels.js

```diff
@@ -88,10 +88,11 @@ function closeSidebarPanel(panelName) {
 
 /**
  * Toggle a sidebar panel from its current state, or set it to the
- * designated `panelState`
+ * designated `panelState` if preset.
```

Comment:
```suggestion
 * designated `panelState`.
```

IMO the sentence reads better without "if preset".